src/shardguard/mcp_servers/run_server.py

```diff
@@ -0,0 +1,81 @@
+import asyncio
+import argparse
+import yaml
+
+from mcp.server import Server
+from mcp.server.stdio import stdio_server
+from mcp.types import Tool, TextContent
+
+print("[DEBUG] run_server.py STARTED")
+
+# Load tool configuration from YAML
+def load_tools_from_yaml(server_name: str, path: str = None) -> list[Tool]:
+    import os
+    if path is None:
+        # Automatically go to root folder where servers.yaml is expected
+        current_file = os.path.abspath(__file__)
+        root_dir = os.path.dirname(current_file)  # Go up to shardguard root
+        path = os.path.join(root_dir, "servers.yaml")
```

Comment:
Why isn't this yaml file a configuration option?   

Why do you assume this is in the root dir?  What's the rationale?

Would it make sense to have a bunch of separate .yaml files (one for each MCP server) instead?